src/vignette/http/middleware.clj

```diff
@@ -41,6 +42,19 @@
 (declare add-cache-control-header
          hours-to-seconds)
 
+(defn log-image-request
+  [handler]
+  (fn [request]
```

Comment:
You could use deconstruction here to get fields from the request, if you want: `{params :query-params uri :uri headers :headers :as request}`. The `:as request` part will still allow you to use the whole argument.